pkg/server/smb/server.go

```diff
@@ -25,6 +29,71 @@ func normalizeWindowsPath(path string) string {
 	return normalizedPath
 }
 
+func normalizeMappingPath(path string) string {
+	items := strings.Split(path, "\\")
+	parts := []string{}
+	for _, s := range items {
+		if len(s) > 0 {
+			parts = append(parts, s)
+			if len(parts) == 2 {
+				break
+			}
+		}
+	}
+	return strings.ToLower("\\\\" + parts[0] + "\\" + parts[1])
+}
+
+func getMappingPathCount(mappingPath string) int {
+	klog.V(2).Infof("calling getMappingPathCount with mapping path %q", mappingPath)
+
+	path := basePath + "/" + strings.Replace(strings.TrimPrefix(mappingPath, "\\\\"), "\\", "/", -1)
```

Comment:
use such way to get <remotePath, localPath> mapping count is quite weird. not sure whether there is better way to achieve this.  cc @jingxu97 @ddebroy

Reply:
Has somebody seen this?